tensorflow_encrypted/protocol/securenn.py

```diff
@@ -106,3 +115,44 @@ def max_pool(self, x):
 
     def dmax_pool_efficient(self, x):
         raise NotImplementedError
+
+
+def _lsb_private(prot: SecureNN, y: PondPrivateTensor):
+    with tf.name_scope('lsb_mask'):
+        with tf.device(prot.crypto_producer.device_name):
+            x = prot.tensor_factory.Tensor.sample_uniform(y.shape)
+            xbits = x.binarize()
+            xlsb = xbits[..., 0]
+            x = PondPrivateTensor(prot, *prot._share(x), is_scaled=True)
```

Comment:
I'd imagine `is_scaled=False` here (since `x` is not a fixedpoint encoding of a value)

Reply:
good catch!  I think this actually raises a larger point -- `is_scaled` is fairly ambiguous here, to me it meant that it's already been implicitly encoded from floating point to an int backing tensor (although in this case there's no actual encoding happening, it's a bit confusing in the general case).